src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -1815,6 +2348,199 @@ public static partial class Kernel32
         [DllImport(nameof(Kernel32), SetLastError = true)]
         public static extern unsafe bool SetInformationJobObject(SafeObjectHandle hJob, JOBOBJECT_INFO_CLASS jobObjectInfoClass, void* lpJobObjectInfo, uint cbJobObjectInfoLength);
 
+        [DllImport(nameof(Kernel32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool GenerateConsoleCtrlEvent(uint dwCtrlEvent, uint dwProcessGroupId);
```

Comment:
Seeing as you went to the trouble to define the `ControlType` type, don't you want to use it instead of `uint` for the first parameter?